views/base.njk

```diff
@@ -136,5 +136,82 @@
 
     {% block main_content %}
     {% endblock %}
+
+    <footer class="usa-footer usa-footer-medium" role="contentinfo">
+        <div class="usa-footer-primary-section">
+            <div class="usa-grid">
+              <nav class="usa-footer-nav">
+                <ul class="usa-unstyled-list">
+                  {% if not isAuthenticated %}
+                    <li class="usa-width-one-sixth usa-footer-primary-content">
+                      <a class="usa-footer-primary-link" href="/">Home</a>
+                    </li>
+                    <li class="usa-width-one-sixth usa-footer-primary-content">
+                      <a class="usa-footer-primary-link" href="/content/examples/">Example sites</a>
+                    </li>
+                    <li class="usa-width-one-sixth usa-footer-primary-content">
+                      <a class="usa-footer-primary-link" target="_blank" href="https://federalist-docs.18f.gov">Documentation</a>
+                    </li>
+                    <li class="usa-width-one-sixth usa-footer-primary-content">
+                      <a class="usa-footer-primary-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSesNI1qlov7Ec_1u2FralYWg9hV4WsB-3FyAHPXc1pRT6In7w/viewform">Get started</a>
+                    </li>
+                  {% else %}
+                    <li class="usa-width-one-sixth usa-footer-primary-content">
+                      <a class="usa-footer-primary-link" href="/">{{ username }}'s sites</a>
+                    </li>
+                    <li class="usa-width-one-sixth usa-footer-primary-content">
+                      <a class="usa-footer-primary-link" target="_blank" href="https://federalist-docs.18f.gov">Documentation</a>
+                    </li>
+                    <li class="usa-width-one-sixth usa-footer-primary-content">
+                      <a class="usa-footer-primary-link" href="https://github.com/18F/federalist/issues/new" target="_blank" rel="noopener noreferrer">File an issue</a>
+                    </li>
+                    <li class="usa-width-one-sixth usa-footer-primary-content">
+                      <a class="usa-footer-primary-link" href="mailto:federalist-support@gsa.gov">Get help</a>
+                    </li>
+                  {% endif %}
+                </ul>
+              </nav>
+            </div>
+        </div>
+        <div class="usa-footer-secondary_section">
+            <div class="usa-grid">
+
+            <div class="usa-footer-contact-links usa-width-one-half">
+
+
+
+              <div class="usa-footer-primary-content usa-footer-contact_info">
+                <p>
+                  <a href="https://github.com/18F/federalist/">
+                    <svg width="22px" height="23px" viewBox="0 0 22 23" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I don't really like them inlined here since it won't be obvious that updating the associated files `icon-envelop.svg` wouldn't be reflected on this page. I think using `img` tags for these might better.

Reply:
That makes sense, I'll get them updated.